client/js/widgets/widget.js

```diff
@@ -723,6 +727,9 @@ export class Widget extends StateManaged {
           c.filter(w=>w.p('type')=='pile').forEach(w=>c.push(...w.children()));
           c = c.filter(w=>w.p('type')!='pile');
           collections[a.collection] = [...new Set(c)];
+
+          if (a.key)
+            this.sortWidgets(collections[a.collection], a.key, a.reverse, a.locales, a.options);
```

Comment:
Can `locales` be an array or why isn't it `locale`?

The function also changes the `z` properties. I don't think a `SELECT` should change the widgets.

Reply:
Right. SELECT shouldn't change the z property.